README.md

```diff
@@ -23,84 +23,149 @@ Dependencies
 Installation
 -----------
 
-1. Download the python-rtmbot code
+1. Create your project
 
-        git clone https://github.com/slackhq/python-rtmbot.git
-        cd python-rtmbot
+        mkdir myproject
+        cd myproject
 
-2. Install dependencies ([virtualenv](http://virtualenv.readthedocs.org/en/latest/) is recommended.)
+2. Install rtmbot (ideally into a virtualenv https://virtualenv.readthedocs.io/en/latest/)
 
-        pip install -r requirements.txt
+        pip install rtmbot
 
-3. Configure rtmbot (https://api.slack.com/bot-users)
+3. Create and rtmbot.conf file and configure rtmbot (https://api.slack.com/bot-users)
```

Comment:
`Create an `rtmbot.conf` file and create a [bot connected to your team](https://api.slack.com/bot-users).`